src/Confluent.Kafka/Consumer.cs

```diff
@@ -671,77 +680,59 @@ public bool Consume(out Message message, int millisecondsTimeout)
             return false;
         }
 
+        /// <summary>
+        ///     Manually consume message or triggers events.
+        ///
+        ///     Will invoke events for OnPartitionsAssigned/Revoked,
+        ///     OnOffsetCommit, etc. on the calling thread.
+        /// </summary>
+        public bool Consume(out Message message, int millisecondsTimeout)
```

Comment:
IMO the `out` parameter [should be ordered last](https://msdn.microsoft.com/en-us/library/ms229015(v=vs.110).aspx). It would be also consistent with other BCL APIs. However I understand it may be a breaking change.